CONTRIBUTING.md

```diff
@@ -1,188 +1,54 @@
 # Contributing to `fastlane`
 
-## Getting started
+## I want to report a problem or ask a question
 
-Make sure you have `bundler` installed using `gem install bundler`
+We use GitHub as support system, please submit a [new issue](https://github.com/fastlane/fastlane/issues/new) to ask questions or to report a problem. 
 
-- Open the GitHub page of the `fastlane`repository  (e.g. [https://github.com/fastlane/fastlane](https://github.com/fastlane/fastlane))
-- Create an issue to discuss your idea/feature/enhancement.
-- Click on `Fork` on the top right
-- Then clone your new repo locally `git clone https://github.com/[my_user]/fastlane.git`
-- On your terminal, navigate to the project and run `git remote add upstream https://github.com/fastlane/fastlane` (or use the `git` URL if you use private key auth)
-- Create your feature branch (`git checkout -b my-new-feature`).
-- Commit your changes (`git commit -am 'Add some feature'`).
-- Push to the branch (`git push origin my-new-feature`).
-- Create a new Pull Request.
+Some people might also use the [fastlane tag on StackOverflow](https://stackoverflow.com/questions/tagged/fastlane), however we don’t actively monitor issues submitted there. 
 
-Before working on something, make sure to have pulled the latest changes. To pull the changes run
+## I want to contribute to fastlane
 
-```
-git pull master
-```
+- To start working on _fastlane_, check out [YourFirstPR.md][firstpr]
+- For some more advanced tooling and debugging tips, check out [ToolingAndDebugging.md](ToolingAndDebugging.md)
 
-## New Issues
+### New Actions
 
-Before submitting a new issue, do the following:
+Please be aware that we don’t accept submissions for new actions at the moment. You can find more information about that [here][submit action].
 
-- Verify you're running the latest version by running `fastlane -v` and compare it with the [project page on GitHub](https://github.com/fastlane/fastlane/tree/master/fastlane).
-- Verify you have Xcode tools installed by running `xcode-select --install`.
-- Make sure to read through the [README](https://github.com/fastlane/fastlane/tree/master/fastlane) of the project.
+## I want to help work on _fastlane_ by reviewing issues and PRs
 
-When submitting a new issue, please provide the following information:
+We are always happy to receive help in that regard by the community! Feel free to read our document on how to [respond to issues and PRs][responding to prs] and also check out how to become a [core contributor][core contributor]. 
```

Comment:
Suggestion:

"We are always happy to receive help in that regard by the community!" -> 

"Thanks! We would really appreciate the help!"